src/main/java/org/folio/services/invoice/InvoiceCancelService.java

```diff
@@ -167,25 +166,25 @@ private Future<Void> unreleaseEncumbrances(List<InvoiceLine> invoiceLines, Invoi
       .filter(InvoiceLine::getReleaseEncumbrance)
       .map(InvoiceLine::getPoLineId)
       .distinct()
-      .collect(toList());
+      .toList();
     if (poLineIds.isEmpty())
       return succeededFuture();
     List<Future<List<PoLine>>> futureList = StreamEx
       .ofSubLists(poLineIds, MAX_IDS_FOR_GET_RQ)
       .map(this::queryToGetPoLinesWithRightPaymentStatusByIds)
       .map(query -> orderLineService.getPoLines(query, requestContext))
-      .collect(toList());
+      .toList();
 
     Future<List<PoLine>> poLinesFuture = collectResultsOnSuccess(futureList)
-      .map(col -> col.stream().flatMap(List::stream).collect(toList()));
+      .map(col -> col.stream().flatMap(List::stream).toList());
 
     return poLinesFuture.compose(poLines -> selectPoLinesWithOpenOrders(poLines, requestContext))
       .compose(poLines -> unreleaseEncumbrancesForPoLines(poLines, invoiceFromStorage, requestContext))
       .recover(t -> {
         Throwable cause = requireNonNullElse(t.getCause(), t);
-        List<Parameter> parameters = Collections.singletonList(
-          new Parameter().withKey("cause").withValue(cause.toString()));
-        Error error = ERROR_UNRELEASING_ENCUMBRANCES.toError().withParameters(parameters);
+        var param = new Parameter().withKey("cause").withValue(cause.toString());
+        String message = String.format(ERROR_UNRELEASING_ENCUMBRANCES.getDescription(), t.getMessage());
+        var error = ERROR_UNRELEASING_ENCUMBRANCES.toError().withMessage(message).withParameters(List.of(param));
         logger.error(error.getMessage(), cause);
```

Comment:
We already have cause in this log, so better to add some message that we can use for quickly search by, because there can be lot of defferent messages
```suggestion
        logger.error("Failed to unrelease encumbrance for po lines", cause);
```

Reply:
Thanks, Changed error message